### ISSUE TITLE
SSLKEYLOGFILE

### ISSUE THREAD

**Issue by crismog** [marten-seemann/qtls#5]
Hello,

I am trying to decrypt quic-go packets with wireshark, but am not able to do this because qtls is not writing TLS secrets to [SSLKEYLOGFILE](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/NSS/Key_Log_Format). Tried this with other quic implementations e.g. ngtcp2 and quicly and works fine.

I can see there is a writeKeyLog() function including all labels in common.go but no secrets are logged to SSLKEYLOGFILE.

Cheers,
Chris

**Comment by marten-seemann** [comment]
That’s correct, I haven’t implemented this.
Do you want to contribute the code to do so?

**Comment by crismog** [comment]
I have changed/added a few (dirty) code lines some time back so that all secrets are logged, but decryption still failed. I then gave up due to lack of time and experience with go language. Here's the diff file if interested: [quic-go_trydecrypt_diff.txt](https://github.com/marten-seemann/qtls/files/3341864/quic-go_trydecrypt_diff.txt).

Maybe I will look into it again in the future. Anyway thanks for your reply.
